charms/kfp-viz/src/charm.py

```diff
@@ -2,73 +2,61 @@
 # Copyright 2021 Canonical Ltd.
 # See LICENSE file for licensing details.
 
+"""Charm the Kubeflow Pipelines Visualization Server.
+
+https://github.com/canonical/kfp-operators
+"""
+
```

Comment:
What is the meaning of `Charm the Kubeflow Pipelines Visualization Server.` ?

Is that an order?
Is there an `of` missing?